x/xos/os_test.go

```diff
@@ -0,0 +1,91 @@
+package xos
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestTouch(t *testing.T) {
+	name := filepath.Join(os.TempDir(), "__test-file")
+	defer Remove(name)
+
+	if err := Touch(name); err != nil {
+		t.Fatalf("Touch failed: %s", err)
+	}
+	if !Exist(name) {
+		t.Fatalf("Touched file dosen't exist")
+	}
+
+	if err := Touch(name); err != nil {
+		t.Fatalf("Touch failed: %s", err)
+	}
+}
+
+func TestExist(t *testing.T) {
+	name := filepath.Join(os.TempDir(), "__test-file")
+	defer Remove(name)
+	if err := Touch(name); err != nil {
+		t.Fatalf("Touch failed: %s", err)
```

Comment:
we can use require package for testing x package

Reply:
I intentionally didn't use require package here to keep it as simple as possible and reduce dependencies.

Reply:
doesn't make sense to me, leaving this to @antho1404's watch

Reply:
I kind of prefer to use the require package in order to have a coherence in the codebase. And for now these packages are still in the project so it doesn't really matter to have this extra dependency but I understand that maybe one day it can be useful.

In other terms I don't mind one or the other are fine, it's not a big deal for me even if I would prefer to use require package from now to have consistency